bin/commands.js

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env node
+const chalk = require('chalk');
+const got = require('got')
+
+const { backupMessages } = require("./backupMessages");
+
+const generateRandomNumber = Math.floor(Math.random() * (backupMessages.length - 1));
+
+
+const getCommitMessage = (prefix) => (async () => {
+	const prefixMessage = chalk.yellowBright('lolcommit: ');
+
+	try {
+		const response = await got('http://whatthecommit.com/index.txt');
+		console.log(prefix ? prefixMessage + response.body : response.body);
+	} catch (error) {
+		console.log(prefix ? prefixMessage + backupMessages[generateRandomNumber] : backupMessages[generateRandomNumber]);
+	}
+})();
+
+const defaultCommand = () => {
```

Comment:
Can this be de-deduped? Default value parameter?

Reply:
[a8d5fa6075a260c2f05df54930758a434b366da6](https://github.com/ellie-keen/lolcommit/pull/11/commits/a8d5fa6075a260c2f05df54930758a434b366da6)